test/integration_test_wrapper.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+'''Test wrapper command for driving integration tests.
+
+Note: This is deliberately rough and only intended to drive existing tests
+with the expectation that as part of formally defining the API it will be tidy.
+
+'''
+
+from argparse import ArgumentParser
+from contextlib import ExitStack
+from logging import getLogger as logging_get_logger, basicConfig as logging_basic_config, DEBUG as LOG_DEBUG
+from pathlib import Path
+from shlex import quote
+from subprocess import DEVNULL, run
```

Comment:
I would suggest we follow the same style as mkosi here and always do `import XXX` for standard library stuff except for `Path` from `pathlib` and types from `typing` and `collections.abc`

Reply:
I might go back to `from shlex import quote` since I use it a lot in f-strings, but otherwise this is fine.